core/com.b2international.snowowl.api.rest/src/com/b2international/snowowl/api/rest/SnowowlSecurityConfig.java

```diff
@@ -68,6 +69,12 @@ protected void configure(final AuthenticationManagerBuilder auth) throws Excepti
 		auth.authenticationProvider(authenticationProvider);
 	}
 	
+	@Override
+	public void configure(WebSecurity web) throws Exception {
+		super.configure(web);
```

Comment:
The method in the superclass is empty; in this case it does not cause problems, but for `configure(HttpSecurity)`, calling it would set up some things that you may not want per the javadoc. I suggest removing this line.